File: atcoder.jp/contests/arc020/tasks/arc020_1/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const [A, B] = input().split(" ").map(Number);

let ans: string;
if (Math.abs(A) < Math.abs(B)) {
  ans = "Ant";
} else if (Math.abs(A) == Math.abs(B)) {
  ans = "Draw";
} else {
  ans = "Bug";
}

console.log(ans);

File: atcoder.jp/contests/arc020/tasks/arc020_1/main.rs
use std::cmp::Ordering;

use proconio::input;

fn main() {
    input! {
        a: i32,
        b: i32,
    }

    let ans = match a.abs().cmp(&b.abs()) {
        Ordering::Less => "Ant",
        Ordering::Equal => "Draw",
        Ordering::Greater => "Bug",
    };

    println!("{}", ans);
}
